### DISCUSSION TITLE
[4.10] certificateCreate form setup

### DISCUSSION THREAD

**Discussion by komalm** [Sunbird-Knowlg/Community#143] [General Discussions]
Hello,

**Site Environments Details** - SunbirdEd portal of version 4.10

**Form Setup -**

Can someone please help us to setup this following form.

```
Request URL: {{host}}/api/data/v1/form/read
Request Method: POST

```

Payload - 

```
request: 
{type: "certificate", action: "certificateCreate", subType: "course", rootOrgId: "*",…}
action: "certificateCreate"
component:"portal"
rootOrgId:"*"
subType:"course"
type:"certificate"
```

@Krishnaj20 Could you please assist us in directing that ticket to the correct person?

**Comment by pallakartheekreddy** [comment]
@komalm Please raise this request in Ed https://github.com/orgs/Sunbird-Ed/discussions

@vinukumar-vs FYI.

**Reply by komalm** [reply]
@pallakartheekreddy Thank you for your response; I've created a new ticket in the Ed.

New Ticket link - https://github.com/orgs/Sunbird-Ed/discussions/573
